src/LibraryManager.Vsix/UI/InstallDialog.xaml

```diff
@@ -159,35 +158,6 @@
         <DockPanel Grid.Row="4" 
                    Grid.ColumnSpan="4" 
                    LastChildFill="False">
-            <StackPanel VerticalAlignment="Top"
-                        HorizontalAlignment="Left"
-                        DockPanel.Dock="Left"
-                        Orientation="Horizontal">
-                <StackPanel.Style>
-                    <Style TargetType="StackPanel">
-                        <Style.Triggers>
-                            <DataTrigger Binding="{Binding Path=AnyFileSelected}" Value="False">
-                                <Setter Property="Visibility" Value="Visible" />
-                            </DataTrigger>
-                            <DataTrigger Binding="{Binding Path=AnyFileSelected}" Value="True">
-                                <Setter Property="Visibility" Value="Hidden" />
-                            </DataTrigger>
-                        </Style.Triggers>
-                    </Style>
-                </StackPanel.Style>
-                <imaging:CrispImage Width="16" 
-                                    Height="16" 
-                                    DockPanel.Dock="Top" 
-                                    HorizontalAlignment="Right">
-                    <imaging:CrispImage.Style>
-                        <Style TargetType="{x:Type imaging:CrispImage}">
-                            <Setter Property="Moniker" Value="{x:Static catalog:KnownMonikers.StatusWarning}" />
-                        </Style>
-                    </imaging:CrispImage.Style>
-                </imaging:CrispImage>
-                <Label Content="{x:Static resources:Text.NoFilesSelected}">
```

Comment:
>NoFilesSelected [](start = 57, length = 15)

I assume these resources are used somewhere else?

Reply:
I'm still using this in the InstallDialogViewModel. IsValidAsync(..) in LibraryInstallationState doesn't check if the files are selected or not. Since UI is the only one doing the check I didn't refactor that bit of code and instead using this(NoFilesSelected) to do the validation.

---
In reply to: [201158225](https://github.com/aspnet/LibraryManager/pull/293#discussion_r201158225) [](ancestors = 201158225)